examples/values.schema.json

```diff
@@ -84,6 +84,15 @@
           "required": [],
           "title": "env"
         },
+        "externalIP": {
+          "default": "443",
```

Comment:
Still quite odd that the default is a string here. It would be fixed by setting `type: integer`, but I get the validation error `cannot use both 'enum' and 'type' in the same schema`, which doesn't really make sense to me. Why can we not use both?